doc/guide.rst

```diff
@@ -0,0 +1,179 @@
+.. _guide_label:
+
+Guide to Simpleline
+===================
+
+Simpleline is the text user interface framework completely written in Python 3 with a possibility to
```

Comment:
`the text user` -> `a text user`

`completely written` -> `written completely`